libs/eo/core/shell/src/lib/eo-shell.component.ts

```diff
@@ -34,7 +33,6 @@ import { EoPrimaryNavigationComponent } from './eo-primary-navigation.component'
   imports: [
     EoFooterComponent,
     EoPrimaryNavigationComponent,
-    MatDateFnsModule,
```

Comment:
Have you tested this doesn't break any functionality in `eo`? Otherwise, please revert the change, and I will prioritize moving `eo` to `dayjs` also :)

Reply:
It will not work with MatDateFnsModule, because angular materiale now uses the native provider and not the date-fns one.
I did launch EO and it looked fine to me, but it might be a good idea with second peer of eyes :-)

Reply:
It looks fine to me :)